src/routes/sessions.js

```diff
@@ -36,8 +35,6 @@ module.exports = function Sessions(app, opts) {
         throw new Error();
       }
 
-      await ipWhitelist.retrieve(envSecret);
```

Comment:
I removed this because:
- it was responsible of a generic toast message "IP Whitelist error" in case of bad env secret
- the ipWhitelist is already retrieved at the server start and at every private route, so no need to add it here.